src/main/java/de/retest/ui/diff/RootElementDifferenceFinder.java

```diff
@@ -78,7 +78,7 @@ private RootElement findAndRemove( final RootElement expected, final List<RootEl
 	}
 
 	private double compareContainedComponents( final RootElement expected, final RootElement bestWindow ) {
-		return getNumberOfChildDifferences( expected, bestWindow )
+		return (double) (getNumberOfChildDifferences( expected, bestWindow ))
 				/ (getNumberOfChildComponents( expected, bestWindow ) + 1);
```

Comment:
You could just juse  `+ 1.0`
```suggestion 
return getNumberOfChildDifferences( expected, bestWindow ) / (getNumberOfChildComponents( expected, bestWindow ) + 1.0);
```